File: Lenguajes Orientados a Objetos/Actividad 11/Java/src/Termometro.java
import java.util.Random;

public class Termometro {
    public float tempC;
    Termometro()
    {
        Random random = new Random();
        tempC = random.nextFloat(100);

    }

    Termometro(float temp)
    {
        tempC = temp;
    }


    public float TemperaturaC()
    {
        System.out.println("La temperatura en °C es de: " + tempC + "°C");
        return tempC;
    }
    public float TemperaturaF()
    {
        float tempF = (tempC * 9/5) + 32f;
        System.out.println("La temperatura en °F es de: " + tempF + "°F");
        return tempF;
    }
    public float TemperaturaK()
    {
        float tempK = tempC + 273.15f;
        System.out.println("La temperatura en K es de: " + tempK + "K");
        return tempK;
    }

    public void MostrarTemperatura()
    {
        this.TemperaturaC();
        this.TemperaturaF();
        this.TemperaturaK();
    }

}

File: Lenguajes Orientados a Objetos/Actividad 11/Python/termometro.py
import random

class Termometro():
    def __init__(self, tempC=random.randint(0, 100)):
        self.tempC = tempC

    def TemperaturaC(self):
        print(f"La temperatura en °C es: {self.tempC}°C ")
        return self.tempC
    
    def TemperaturaK(self):
        tempK = self.tempC + 273.15
        print(f"La temperatura en K es: {tempK}K ")
        return tempK
    
    def TemperaturaF(self):
        tempF = (self.tempC*9/5) + 32
        print(f"La temperatura en °F es: {tempF}°F ")
        return tempF
    
    def MostrarTemperatura(self):
        self.TemperaturaC()
        self.TemperaturaF()
        self.TemperaturaK()